.github/workflows/canary-integration-test.yml

```diff
@@ -842,6 +842,7 @@ jobs:
         run: |
           yq write -i deploy/examples/operator.yaml "data.ROOK_CEPH_ALLOW_LOOP_DEVICES" --style=double "true"
           tests/scripts/github-action-helper.sh deploy_manifest_with_local_build deploy/examples/operator.yaml
+          tests/scripts/github-action-helper.sh deploy_manifest_with_local_build deploy/examples/csi-operator.yaml
```

Comment:
The `encryption-pvc`, `pvc`, and `pvc-db-wal` tests are all hanging during uninstall waiting for the csi operator's `clientprofile` CR to be deleted. The csi operator is getting deleted before it can remove the finalizer from that CR. So these tests will need to delete the clientprofile CR first, before continuing with other cleanup.